homeassistant/components/fluss/button.py

```diff
@@ -0,0 +1,60 @@
+"""Support for Fluss Devices."""
+
+import logging
+
+from fluss_api.main import FlussApiClient
+
+from homeassistant.components.button import ButtonEntity
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+LOGGER = logging.getLogger(__package__)
+DEFAULT_NAME = "Fluss +"
+
+
+async def async_setup_entry(
+    hass: HomeAssistant, entry: ConfigEntry, async_add_entities: AddEntitiesCallback
+) -> None:
+    """Set up the Fluss Devices."""
+
+    api: FlussApiClient = entry.runtime_data
+
+    devices_data = await api.async_get_devices()
+    devices = devices_data["devices"]
+
+    buttons = [
+        FlussButton(api, device) for device in devices if isinstance(device, dict)
+    ]
+    LOGGER.debug("Adding entities: %s", buttons)
+    async_add_entities(buttons)
```

Comment:
```suggestion
    async_add_entities(FlussButton(api, device) for device in devices if isinstance(device, dict))
```
imo there's no point to log this, as you'll see the buttons you get